internal/service/docdb/sweep.go

```diff
@@ -23,6 +23,212 @@ func init() {
 			// "aws_docdb_cluster",
 		},
 	})
+	resource.AddTestSweepers("aws_docdb_subnet_group", &resource.Sweeper{
+		Name:         "aws_docdb_subnet_group",
+		F:            sweepDBSubnetGroups,
+		Dependencies: []string{},
+	})
+	resource.AddTestSweepers("aws_docdb_event_subscription", &resource.Sweeper{
+		Name:         "aws_docdb_event_subscription",
+		F:            sweepEventSubscriptions,
+		Dependencies: []string{},
+	})
+	resource.AddTestSweepers("aws_docdb_cluster", &resource.Sweeper{
+		Name:         "aws_docdb_cluster",
+		F:            sweepDBClusters,
+		Dependencies: []string{},
+	})
+	resource.AddTestSweepers("aws_docdb_cluster_snapshot", &resource.Sweeper{
+		Name:         "aws_docdb_cluster_snapshot",
+		F:            sweepDBClusterSnapshots,
+		Dependencies: []string{},
+	})
+	resource.AddTestSweepers("aws_docdb_cluster_instance", &resource.Sweeper{
+		Name:         "aws_docdb_cluster_instance",
+		F:            sweepDBInstances,
+		Dependencies: []string{},
+	})
+	resource.AddTestSweepers("aws_docdb_cluster_parameter_group", &resource.Sweeper{
+		Name:         "aws_docdb_cluster_parameter_group",
+		F:            sweepDBClusterParameterGroups,
+		Dependencies: []string{},
+	})
+}
+
+func sweepDBClusters(region string) error {
+	client, err := sweep.SharedRegionalSweepClient(region)
+
+	if err != nil {
+		return fmt.Errorf("error getting client: %w", err)
+	}
+
+	conn := client.(*conns.AWSClient).DocDBConn
+	input := &docdb.DescribeDBClustersInput{}
+
+	err = conn.DescribeDBClustersPages(input, func(out *docdb.DescribeDBClustersOutput, lastPage bool) bool {
+		for _, dBCluster := range out.DBClusters {
+			id := aws.StringValue(dBCluster.DBClusterIdentifier)
+			input := &docdb.DeleteDBClusterInput{
+				DBClusterIdentifier: dBCluster.DBClusterIdentifier,
+			}
```

Comment:
This will need to skip creating the final snapshot, or you'll get the error

> InvalidParameterCombination: Please specify FinalDBSnapshotIdentifier or SkipFinalSnapshot

```suggestion
				SkipFinalSnapshot: true,
			}
```